### PR TITLE
Fix links in contributing.md

### PR DESCRIPTION
**Pull request by kbressem**
Fixes links in CONTRIBUTING.md.

### Description
I found a few dead links in CONTRIBUTING.md and corrected them

### Status
**Ready**

### Please ensure all the checkboxes:
**no code changes**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ pathology_metastasis_detection
 
 ### Model storage
 
-Github limits the size of files allowed in the repository (see [About size limits on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)). Therefore, MONAI Model Zoo limits each single file to be no larger than **25MB**.
+Github limits the size of files allowed in the repository (see [About size limits on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github). Therefore, MONAI Model Zoo limits each single file to be no larger than **25MB**.
```
Comment (Nic-Ma) [inline CONTRIBUTING.md]: I think this ")" is to match the "(" in "... allowed in the repository (see ...".

Thanks.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -117,7 +117,7 @@ All code changes to the dev branch must be done via [pull requests](https://help
 1. [create a new branch in your fork](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request-from-a-fork)
 of the codebase named `[ticket_id]-[task_name]`.
 Ideally, the new branch should be based on the latest `dev` branch.
-1. Make changes to the branch ([use detailed commit messages if possible](https://chris.beams.io/posts/git-commit/)).
+1. Make changes to the branch ([use detailed commit messages if possible](https://chris.beams.io/posts/git-commit/).
```
Comment (Nic-Ma) [inline CONTRIBUTING.md]: Same comment as above.

Thanks.

File: CONTRIBUTING.md (hunk 1 above)
Comment (kbressem) [reply CONTRIBUTING.md]: Yes, you are correct. I've reverted these changes.

**Comment by kbressem** [comment]
btw, in the README, the following link does not work as well: 

MONAI Bundle demo: https://github.com/Project-MONAI/tutorials/tree/main/model_zoo 

But here I can't figure out the correct link.

**Review by Nic-Ma — APPROVED** [review]

**Comment by Nic-Ma** [comment]
Hi @kbressem ,

The model_zoo tutorial depends on this PR: https://github.com/Project-MONAI/tutorials/pull/873

Thanks.

**Comment by kbressem** [comment]
> Hi @kbressem ,
> 
> The model_zoo tutorial depends on this PR: [Project-MONAI/tutorials#873](https://github.com/Project-MONAI/tutorials/pull/873)
> 
> Thanks.

Thank you for the clarification @Nic-Ma . I guess the PR will be done soon, so the link will become valid then.

**Pull request merged**